logger.js

```diff
@@ -12,8 +12,9 @@ function pinoLogger (opts, stream) {
   opts.serializers = opts.serializers || {}
   opts.serializers.req = opts.serializers.req || asReqValue
   opts.serializers.res = opts.serializers.res || pino.stdSerializers.res
+  opts.stream = opts.stream || stream
```

Comment:
the problem is that we are always passing `opts.stream`  property inside opts. This will be picked up by pino.

We should do:

```js
var stream = opts.stream || stream
delete opts.stream
```